### ISSUE TITLE
generate distractors

### ISSUE THREAD

**Issue by DivyaWadehra** [harrylclc/LTR-DG#5]
How to generate distractors, if we just have question and answer

**Comment by jacampo** [comment]
Hello @DivyaWadehra Did you do it? That could be really helpful

**Comment by DivyaWadehra** [comment]
@jacampo , Yes I have done some research and coding on the same. Will publish it soon in my github repo 
[https://github.com/DivyaWadehra/Intelligent-Multiple-choice-Questions-and-Answers-Generation](https://github.com/DivyaWadehra/Intelligent-Multiple-choice-Questions-and-Answers-Generation)

**Comment by jacampo** [comment]
@DivyaWadehra Nice! I will take a look. I´m looking for ideas to do an aproach in spanish and this can be helpful for the end.
